src/context.ts

```diff
@@ -64,3 +66,19 @@ export class OperonContextImpl implements OperonContext {
     return get(this.applicationConfig, key);
   }
 }
+
+export interface InitContext extends OperonContext {
+  readonly userDatabase: UserDatabase;
```

Comment:
I don't think we want to directly expose the userDatabase object -- probably we just need some methods to create/drop schema, similar to the ones in testing runtime.

Reply:
Do we not give people the userDatabase any other time?